Automattic/jetpack
# Cookie consent: remove fixed position in the editor

## Proposed changes:
Currently, the cookie consent block has `position: fixed` in the editor, but it's corrected by the following core styles

```css
.block-editor-block-list__layout .block-editor-block-list__block {
    overflow-wrap: break-word;
    pointer-events: auto;
    position: relative;
    -webkit-user-select: text;
    user-select: text
}
```

In some themes, these styles don't exist, and the block is rendered with `position: fixed`.  This PR moves the `position: fixed` to the frontend only.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
1. Go to the Site Editor (not post editor).
2. Add the cookie consent block.
3. It shouldn't be absolute.
4. To visit the site, the cookie consent block should be fixed.
